.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       with:
         name: ic-hs-test
         # authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-    - run: git rm motoko
+    - run: cd motoko && git submodule deinit
```

Comment:
I just did a `mv .git .git-junk`

Reply:
Just saw this after trying something similar, will switch to this if the `rm -rf` causes any issues.